plugin.video.giantbomb/default.py

```diff
@@ -107,7 +107,7 @@ def list_categories():
 
     live_data = gb.query('video/current-live')
 
-    if live_data['video'] is not None:
+    if live_data.get('video', None) is not None:
```

Comment:
The `, None` isn't necessary, since `dict.get()` returns `None` by default when the key doesn't exist:

```suggestion
    if live_data.get('video') is not None:
```